### DISCUSSION TITLE
Use more screen width.

### DISCUSSION THREAD

**Discussion by krischik** [jpanther/congo#668] [Q&A] [answered]
I have some larger tables to display and they would benefit from more screen width. Is there a way to reduce the ample left and right margins?

**Comment by krischik** [comment]
So, I guess it's not possible to adjust the screen width. I guess i have to look for another theme then. Having my tables squeezed to an ugly blob while at the same time 40% of the screen is empty is not helpful at all.

**Reply by jpanther** [reply]
Have you tried using the `simple` layout? This allows full-width content pages. You can read more about this [in the docs](https://jpanther.github.io/congo/docs/content-examples/#simple-pages).

**Reply by krischik** [reply]
No I have not. All the documentation [I read until now](https://jpanther.github.io/congo/docs/homepage-layout/) only had the options `page` and `profile`. Now I see there are more options.

**Comment by wolfspyre** [comment]
in custom.css:

‘.max-w-prose { max-width: 80ch !important;}’

**Reply by krischik** [reply]
I don't yet have a custom.css — which directory does it go?

**Reply by wolfspyre** [reply, marked as answer]
it’s not my intention to sound like a jerk. This is documented quite well, however.

Your experience with the theme will likely be much more enjoyable if you peruse the documentation; I’d be willing to bet several questions you have are already answered.

specifically: https://jpanther.github.io/congo/docs/advanced-customisation/#overriding-the-stylesheet

**Reply by krischik** [reply]
I did try the google search and didn't find it and I click through the documentation all the time. But there are still so many questions. I did notice that `/content/_index.md` makes better use of the screen estate. I also noticed that some pages in the sample have side bar menus. However, my pages never have side bar menus.

I don't really need the side bar menus. If I could trick the system to always use the layout from `/content/_index.md`…

Yes, I can: `.max-w-prose { max-width: 100% !important;}`

**Reply by wolfspyre** [reply]
so a simple solution would be to take the template for the page-type you use the most, and override it by placing a copy of it in your site's `layouts/partials` directory ... this overrides that partial template with YOUR version.... then simply change the associated classes that object is decorated with. 

easy peasy ;)

**Comment by wolfspyre** [comment]
in light of this, might it be worth adding a knob to adjust this? its a reasonable (imo) ask. maybe a tailwind breakpoint for window size? or just have a param to adjust?

most browser dev tools have a way to inspect a particular element to see what styling is controlling an element….  from there it’s relatively easy to identify why a particular element has… or doesn’t have… styling that is undesirable… 

this is roughly what I’ve been doing…. so if someone has a better way to manipulate and refine the styling here, it’d be cool to try and find the cleanest way to consistently adjust stuff, so we don’t all end up inventing our own friction reducing and traction increasing rotational cushion bearings …. :) ….
